statsmodels/stats/tests/test_diagnostic.py

```diff
@@ -167,13 +167,13 @@ def test_hac(self):
 
         cov = sw.cov_hac_simple(res, nlags=4, use_correction=False)
         bse_hac = sw.se_cov(cov)
-        assert_almost_equal(cov, cov_hac_4, decimal=14)
+        assert_almost_equal(cov, cov_hac_4, decimal=13)
         assert_almost_equal(bse_hac, np.sqrt(np.diag(cov)), decimal=14)
 
         cov = sw.cov_hac_simple(res, nlags=10, use_correction=False)
         bse_hac = sw.se_cov(cov)
-        assert_almost_equal(cov, cov_hac_10, decimal=14)
-        assert_almost_equal(bse_hac, np.sqrt(np.diag(cov)), decimal=14)
+        assert_almost_equal(cov, cov_hac_10, decimal=12)
+        assert_almost_equal(bse_hac, np.sqrt(np.diag(cov)), decimal=12)
```

Comment:
do you have an idea why the precision here is lower?

Is the dtype, float64, for the data the same as before?

Reply:
I think this might be fixed by passing “float_precision=‘high’” to pd.read_csv